plugins/alerting/pkg/alerting/api_conditions.go

```diff
@@ -574,3 +575,39 @@ func (p *Plugin) Timeline(ctx context.Context, req *alertingv1.TimelineRequest)
 
 	return resp, nil
 }
+
+func (p *Plugin) CloneTo(ctx context.Context, req *alertingv1.CloneToRequest) (*emptypb.Empty, error) {
+	lg := p.Logger.With("handler", "CloneTo")
+	if err := req.Validate(); err != nil {
+		return nil, err
+	}
+	clusterLookup := map[string]struct{}{}
+	cl, err := p.mgmtClient.Get().ListClusters(ctx, &managementv1.ListClustersRequest{})
+	if err != nil {
+		return nil, err
+	}
+	for _, c := range cl.Items {
+		clusterLookup[c.Id] = struct{}{}
+	}
+	for _, ref := range req.ToClusters {
+		if _, ok := clusterLookup[ref]; !ok {
+			return nil, validation.Errorf("cluster could not be found %s", ref)
+		}
+	}
+	iErrGroup := &independentErrGroup{}
```

Comment:
Errors are not being added to the error group here?

Reply:
Added